features/element-hiding.json

```
@@ -2173,6 +2173,22 @@
                     }
                 ]
             },
+            {
+                "domain": "lowes.com",
+                "rules": [
+                    {
+                        "type": "disable-default"
+                    },
+                    {
+                        "selector": "[class*='__GAMWrapper-']",
+                        "type": "override"
+                    },
+                    {
+                        "selector": ".gam-wrapper'",
```

```suggestion
                        "selector": ".gam-wrapper",
```
There’s an extra `'` in this line. Without that, it should be fine.